core/src/cats/ddl/creates/postgresql.sql

```diff
@@ -597,6 +597,75 @@ WHERE (db_mb IS NOT NULL AND db_mb > 0)
    OR (vm_mb IS NOT NULL AND vm_mb > 0)
 ;
 
-
+-- lstat_decode part
+-- This function emulate in plpgsql language the C++ code of FromBase64 function in core/src/lib/base64.cc
+create or replace function bareos_frombase64(field text) returns bigint as $$
+ declare
+  i integer default 0;
+  res bigint default 0;
+  rf text default '';
+  base64 text default 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/';
+  z integer := char_length(field); -- minus 1s
```

Comment:
what does that comment mean?

Reply:
good catch I forget to remove them. will do